modules/misc/version.nix

```diff
@@ -5,7 +5,8 @@ with lib;
 {
   options = {
     home.stateVersion = mkOption {
-      type = types.enum [ "18.09" "19.03" "19.09" "20.03" "20.09" "21.03" ];
+      type =
+        types.enum [ "18.09" "19.03" "19.09" "20.03" "20.09" "21.03" "21.05" ];
```

Comment:
I suppose all the logic against "21.03" will have to support "21.05" too (and vice-versa)?